starter-code/index.html

```diff
@@ -3,40 +3,85 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="./css/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet">
 </head>
+
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+<nav>
+  <img id="logo-spotify" src="./images/spotify-logo.png">
+  <ul>
+    <li>Premium</li>
+    <li>Discover</li>
+    <li>Help</li>
+    <li>Download</li>
+  </ul>
+</nav>
+</div>
+  
+<div id="landing">
+    <img src="./images/landing.jpg">
+    <h1>Music for everyone.</h1> 
+    <p id="title">Spotify is now free on mobile, tablet and computer.<br/>
+      Listen to the right music, wherever you are.</p>
+</div>
+
+<div id="white-background">
+  <h3>What’s on Spotify?</h3>
+        <div>
```

Comment:
add the same class for each of the three div tags, then style them adding some width and floating them. (now you know flexbox)